TACC/Core-Styles
# fix(bs3): plain breadcrumb has padding-top

## Overview

Remove extra padding-top from BS3 breadcrumb.

## Related

- fixes #346

## Changes

- **removes** padding-top from bs3 plain breadcrumb

## Testing

1. Open http://localhost:3000/components/detail/bootstrap3--breadcrumb.
2. Verify "Plain" `.breadcrumb` has **no** `padding-top`.

## UI

| | Before | After |
| - | - | - |
| BS3 | <img width="1200" alt="bs3 before" src="https://github.com/TACC/Core-Styles/assets/62723358/ed6db607-ae3f-4146-9535-bdb86c63e498"> | <img width="1200" alt="bs3 after" src="https://github.com/TACC/Core-Styles/assets/62723358/c5d99147-179a-4769-ac59-0ffbacc2b7f8"> |
| TACC | <img width="1200" alt="tacc before" src="https://github.com/TACC/Core-Styles/assets/62723358/5433a906-d1ae-498f-adec-8d3c9ca03064"> | <img width="1200" alt="tacc after" src="https://github.com/TACC/Core-Styles/assets/62723358/c450e434-b91f-41d9-87de-37a5c46c1b40"> |
